python/requirements/nsight_mock/nsys_mock.py

```diff
@@ -0,0 +1,32 @@
+import argparse
```

Comment:
technically, this is a fake, not a mock

a fake is an implementation that satisfies the interface but does not do the real thing.

a mock is a test setup that intercepts and monkey patches certain function calls or logic invocations.